hack/cherry_pick_pull.sh

```diff
@@ -16,7 +16,8 @@
 
 # Checkout a PR from GitHub. (Yes, this is sitting in a Git tree. How
 # meta.) Assumes you care about pulls from remote "upstream" and
-# checks thems out to a branch named pull_12345.
+# checks thems out to a branch named:
+#  automated-cherry-pick-of-<pr>-<target branch>-<timestamp>
```

Comment:
What does it do if that branch already exists? (like it would if you ran it with dry run and then ran it again)